assets/src/design-system/components/mediaInput/index.js

```diff
@@ -40,8 +40,8 @@ import { ReactComponent as Landscape } from './landscape.svg';
 import { MEDIA_VARIANTS } from './constants';
 
 const MediaRectangle = styled.section`
-  width: 64px;
-  height: 114px;
+  width: 100%;
+  height: 100%;
```

Comment:
Remember to update the sizing in Storybook - looks really weird now:

<img width="671" alt="Screen Shot 2021-03-15 at 20 59 33" src="https://user-images.githubusercontent.com/637548/111240382-7dca4880-85d1-11eb-991f-ecbd449ced0b.png">